homeassistant/components/govee_light_local/coordinator.py

```diff
@@ -32,32 +34,45 @@ def __init__(self, hass: HomeAssistant) -> None:
             update_interval=SCAN_INTERVAL,
         )
 
-        self._controller = GoveeController(
-            loop=hass.loop,
-            logger=_LOGGER,
-            broadcast_address=CONF_MULTICAST_ADDRESS_DEFAULT,
-            broadcast_port=CONF_TARGET_PORT_DEFAULT,
-            listening_port=CONF_LISTENING_PORT_DEFAULT,
-            discovery_enabled=True,
-            discovery_interval=CONF_DISCOVERY_INTERVAL_DEFAULT,
-            discovered_callback=None,
-            update_enabled=False,
-        )
+        self._controllers = []
+        for source_ip in source_ips:
```

Comment:
Another candidate for list comprehension.

Reply:
done.